dsl\qualifier.c(31,1): error C2059: syntax error: '}'

Using `Visual Studio 2017 Build Tools` and `Visual Studio 2019 Build Tools` to build `dsl\qualifier.c` reports following:

```
(ClCompile target) ->
  D:\a\cmakew\ctags\dsl\qualifier.c(31,1): error C2059: syntax error: '}' [D:\a\cmakew\ctags\build.d\readtags.vcxproj]
```

relavent code:[dsl/qualifier.c#L30-L31](https://github.com/universal-ctags/ctags/blob/master/dsl/qualifier.c#L30-L31)

```
static DSLProcBind pbinds [] = {
};
```

change to following can resolve this problem:

```
static DSLProcBind pbinds [] = {0};
```

Note: `dsl\qualifier.c` is not used on Windows building with `MSVC`, this problem has been reported as #3005

--- a/dsl/qualifier.c
+++ b/dsl/qualifier.c
@@ -27,8 +27,6 @@ struct sQCode
 /*
  * DATA DEFINITIONS
  */
-static DSLProcBind pbinds [] = {
-};
 
 
 /*
@@ -41,7 +39,7 @@ static int initialize (void)
 	if (initialized)
 		return 1;
 
-	if (!dsl_init (DSL_QUALIFIER, pbinds, sizeof(pbinds)/sizeof(pbinds [0])))
+	if (!dsl_init (DSL_QUALIFIER, NULL, 0))
 	{
 		fprintf(stderr, "MEMORY EXHAUSTED\n");
 		return 0;

